CMFS_20NG.ipynb

```diff
@@ -114,6 +115,20 @@
    ]
   },
   {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# New Matrix after applying Variance. \n",
+    "selector = VarianceThreshold()\n",
+    "selector.fit(term_category_mat)\n",
+    "selector.fit_transform(term_category_mat)"
```

Comment:
The output from `fit_transform` is not being used anywhere. This could be stored back in `term_category_mat`.